.github/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Community engineering issue
+about: Create an issue related to a community engineering effort
+---
+
+# Community engineering issue
+
+<!-- (REQUIRED) Provide information for the doc request including any Community code issues or PRs, Magento versions, or user guide pages. -->
+
+Type of Doc:
+
+-  [ ] New topic
+-  [ ] Topic Update, user guide link:
+-  [ ] Other
+
+Magento 2 GitHub Issue: https://github.com/magento/merchdocs/issues/link
```

Comment:
it is not clear from the context how the link `https://github.com/magento/merchdocs/issues/link` relates to the text `Magento 2 GitHub Issue: `

Reply:
I added from the devdocs example but replaced with merchdocs. I think they should reference issue or PR in Community repo probably, do you agree?